core/src/global.css

```diff
@@ -21,6 +21,10 @@ html:not(.x) {
 	--gray-8: hsl(216, 78%, 15%); /* 700 */
 	--gray-9: hsl(216, 80%, 10%);
 
+	--support-blue-0: hsl(211, 100%, 90%);
+	--support-blue-1: hsl(221, 100%, 61%);
+	--support-blue-2: hsl(217, 85%, 49%);
+
```

Comment:
There are 2 things here:
1. Other colors in "support" won't be semantic colors, but only category colors, so I think we're good to just use "--support" or "--secondary" here
2. A color range lies from -0 to -9, so try to match these values to those scale. You don't need to provide the middle items

```suggestion
	--support-1: hsl(211, 100%, 90%);
	--support-3: hsl(221, 100%, 61%);
	--support-4: hsl(217, 85%, 49%);
```